src/client/Client.cc

```diff
@@ -6650,10 +6650,18 @@ void Client::_unmount(bool abort)
 
   mount_cond.wait(lock, [this] {
     if (!mds_requests.empty()) {
```

Comment:
If we return from here if the `mds_requests` is non-empty, then the below code would not be executed, also the removal of line `return mds_requests.empty();` indicates this line should be `if(mds_requests.empty())`?

Reply:
Yeah, you are right. It should be `if(mds_requests.empty())`, I will fix it. thanks @dparmar18